validations/src/validations.rs

```diff
@@ -1280,6 +1340,53 @@ pub fn validate_block(
     }
 }
 
+/// Validate a genesis block: a block with hash_prev_block = bootstrap_hash
+pub fn validate_genesis_block(genesis_block: &Block) -> Result<(), BlockError> {
+    // Do not check VRF proof or signatures, the hash is enough proof
+    // But check that there is no extra information in the unused fields
+    let txns = BlockTransactions {
+        mint: MintTransaction::default(),
+        value_transfer_txns: genesis_block.txns.value_transfer_txns.clone(),
+        data_request_txns: vec![],
+        commit_txns: vec![],
+        reveal_txns: vec![],
+        tally_txns: vec![],
+    };
+
+    let merkle_roots = BlockMerkleRoots {
+        mint_hash: txns.mint.hash(),
+        vt_hash_merkle_root: merkle_tree_root(&txns.value_transfer_txns),
+        dr_hash_merkle_root: merkle_tree_root(&txns.data_request_txns),
+        commit_hash_merkle_root: merkle_tree_root(&txns.commit_txns),
+        reveal_hash_merkle_root: merkle_tree_root(&txns.reveal_txns),
+        tally_hash_merkle_root: merkle_tree_root(&txns.tally_txns),
+    };
+    let empty_block = Block {
```

Comment:
it is not an empty block, should we use another name?

Reply:
`block_with_most_fields_set_to_the_default_value` doesn't sound quite good to me :laughing: